### ISSUE TITLE
Write exception report

### ISSUE THREAD

**Issue by bdw429s** [foundeo/toscript#31]
Would be a nice feature of the CLI to write out a report file with all the errors encountered.  They're in the output right now, but since there's so much other (yellow) text they can get lost, especially once the console is closed.

An exception report would let the user know what files to go back and finalize.

**Comment by pfreitag** [comment]
Good idea. It might be also be good to add some stats at the bottom like:

`15 Errors - 128 Files Converted - 8 Files already Script`
